apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx

```diff
@@ -56,16 +56,17 @@ interface SignInMethodLandingPropsInterface extends SBACInterface<FeatureConfigI
 /**
  * Landing component for Application Sign-in method configurations.
  *
- * @param {SignInMethodLandingPropsInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
```

Comment:
Can we remove the redundant line

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3440/commits/d16981abb35182947ec076270163483cb6bd55f6